sdk/communication/Azure.Communication.Administration/README.md

```diff
@@ -23,7 +23,7 @@ dotnet add package Azure.Communication.Administration --version 1.0.0-beta.3
 
 You need an [Azure subscription][azure_sub] and a [Communication Service Resource][communication_resource_docs] to use this package.
 
-To create a new Communication Service, you can use the [Azure Portal][communication_resource_create_portal] or the [.NET management client library][communication_resource_create_net].
+To create a new Communication Service, you can use either the [Azure Portal][communication_resource_create_portal], or the [Azure PowerShell][communication_resource_create_power_shell], or the [.NET management client library][communication_resource_create_net].
```

Comment:
I think saying "you can use the Azure Portal, the Azure PowerShell or the .NET library.." seems more readable, no?

Reply:
Thanks Ankit, I've updated accordingly